src/index.js

```
@@ -0,0 +1,89 @@
+/**
+ * Gmail Invoice PDF Collector - Entry Point
+ * 
+ * This application scans emails, identifies invoices using an LLM,
+ * downloads PDF attachments, and merges them into a single file.
+ */
+const path = require('path');
+const { createConfig, defaultConfig } = require('../config');
```

ESLint

[error] 'defaultConfig' is assigned a value but never used.

(no-unused-vars)